### DISCUSSION TITLE
有人会遇到一些奇怪的编译报错么？感觉taro很难用，动不动就有个奇怪的编译错误，而且一点报错提示信息都没有。

### DISCUSSION THREAD

**Discussion by Sean-lxy** [NervJS/taro#15935] [Q & A]
<img width="1095" alt="image" src="https://github.com/NervJS/taro/assets/10398990/4f921a1f-a27b-473e-b437-108edfa992a9">
<img width="764" alt="image" src="https://github.com/NervJS/taro/assets/10398990/6f667356-c9d6-4432-a431-6d97bd88b412">

**Comment by honlyHuang** [comment]
看你的代码确实没看出哪里有问题，你排查到了没？

**Comment by thomas-void0** [comment]
这个框架问题多的一比，有时候两次编译的结果都不同，好好的代码跑不动了。哈哈哈哈
